nerves_defconfig

```diff
@@ -49,10 +50,18 @@ BR2_PACKAGE_FWUP=y
 BR2_PACKAGE_LINUX_FIRMWARE=y
 BR2_PACKAGE_RPI_FIRMWARE=y
 BR2_PACKAGE_RPI_FIRMWARE_X=y
+BR2_PACKAGE_DBUS_GLIB=y
 BR2_PACKAGE_RPI_USERLAND=y
+BR2_PACKAGE_SETSERIAL=y
 BR2_PACKAGE_ERLANG_SMP=y
 BR2_PACKAGE_LIBMNL=y
-BR2_PACKAGE_NCURSES=y
+BR2_PACKAGE_BLUEZ5_UTILS=y
+BR2_PACKAGE_BLUEZ5_UTILS_OBEX=y
+BR2_PACKAGE_BLUEZ5_UTILS_CLIENT=y
+BR2_PACKAGE_BLUEZ5_UTILS_GATTTOOL=y
+BR2_PACKAGE_BLUEZ5_UTILS_EXPERIMENTAL=y
+BR2_PACKAGE_DNSMASQ=y
```

Comment:
Regarding `setserial`, the `BR2_USE_MMU` indicates that `setserial` requires an MMU to work (the Buildroot Config.in file says that it calls `fork()`, and `fork()` doesn't work on the MMU-less ucLinux. `setserial` is a utility program for configuring serial ports. It shouldn't be needed unless there's a shell script somewhere that calls into it. My suspicion is that `setserial` was enabled to help debug a serial port issue and is no longer needed.